database-ms/src/main/java/org/dice_research/sask/database_ms/RDFTriples/TripleDTO.java

```diff
@@ -0,0 +1,22 @@
+/**

```

Comment:
What does TDO stand for?

Reply:
DTO stands for Data transfer object. I wanted to keep the structure the same as other ms so I named it tripleDTO.